var/spack/repos/builtin/packages/qgis/package.py

```diff
@@ -127,6 +132,9 @@ class Qgis(CMakePackage):
     depends_on("proj@4.9.3:", when="@3.8.2:")
     depends_on("proj@7.2:", when="@3.28:")
     depends_on("proj@:8", when="@3.28")  # build fails with proj@9
+    # fails to build with proj 9.4+ until the backported patch in 3.34.5
+    # https://github.com/qgis/QGIS/compare/final-3_34_4%5E...final-3_34_5
```

Comment:
It's a bit hard to parse this long list of commits, I would personally link to one or both of:

* https://github.com/qgis/QGIS/pull/56761
* https://github.com/qgis/QGIS/pull/56765

Reply:
It took me a while to dig through what release this was in, but noting the release and then linking to https://github.com/qgis/QGIS/pull/56761 makes sense to me